recipes/tdameritrade/meta.yaml

```diff
@@ -11,24 +11,32 @@ source:
 
 build:
   number: 0
-  skip: True  # [py<37 or (win and vc<14)]
+  noarch: python
   script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - tdameritrade-auth = tdameritrade.auth:main
 
 requirements:
   host:
-    - python
+    - python >=3.7
     - pip
   run:
-    - python
+    - python >=3.7
     - ipython >=7.0.1
-    - {{ pin_compatible('numpy') }}
```

Comment:
I think you need to add `numpy` in place of the `pin`

Reply:
I don't actually use bumpy directly so I figured I could rely on pandas